root/user/ReportUser.js

```diff
@@ -128,20 +128,19 @@ const ReportUser = ({
         rows={10}
       />
 
-      <FormRow hasNoLabel>
-        <p>
-          {exp.l(
-            `If you don’t want our admins to contact you further regarding 
-             this report, you can uncheck the checkbox below.
-             <br />
-             We recommend leaving it checked, so that you can be contacted if 
-             the report is resolved or the admins need more information.`,
-          )}
-        </p>
-      </FormRow>
-
       <FormRowCheckbox
         field={form.field.reveal_address}
+        help={
+          <p>
+            {exp.l(
+              `If you don’t want our admins to contact you further regarding 
+              this report, you can uncheck the checkbox above.
```

Comment:
Heh, good to see you did `s/below/above/` here already (noticed it was still "below" in the screenshot). Though the indentation is off by one:
```suggestion
               this report, you can uncheck the checkbox above.
```